nixos/tests/ejabberd.nix

```diff
@@ -0,0 +1,303 @@
+import ./make-test.nix {
+  name = "ejabberd";
+  nodes = {
+    client = { nodes, pkgs, ... }: {
+      environment.systemPackages = let
+        sendMessage = pkgs.writeScriptBin "send-message" ''
+          #!/usr/bin/env python3
```

Comment:
```suggestion
          #! ${(pkgs.python3.withPackages (ps: [ ps.sleekxmpp ])).interpreter}
```

Reply:
This python script is lifted from the prosody test, so this should probably be fixed there, as well.

Reply:
It's not a necessity because both ways work fine. However, it would be nice to have a single file for the script.

Reply:
How about something like this?